manifests/k/KiCad/KiCad/5.1.12_1/KiCad.KiCad.installer.yaml

```diff
@@ -0,0 +1,27 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: KiCad.KiCad
+PackageVersion: 5.1.12_1
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://kicad-downloads.s3.cern.ch/windows/stable/kicad-5.1.12_1-x86_64.exe

```

Comment:
```suggestion
  InstallerUrl: https://osdn.net/frs/redir.php?m=auto&f=%2Fstorage%2Fg%2Fk%2Fki%2Fkicad%2Fkicad-5.1.12_1-x86_64.exe
```

Reply:
Worldwide link so people in e.g. China don't get slow download speeds from EU mirror.

Validation-Domain needs to be taken a look by an engineer.

Reply:
I was unable to download the full version from this mirror, therefore I choose the next best source.